ReadMe.md

```diff
@@ -1 +1,6 @@
-This is our first ReadMe
```

Comment:
Maybe move this to a requirements.txt file? So that the packages can just be installed with `pip install -r requirements.txt`